frontend/src/components/chrome/drawer-manage-network.tsx

```diff
@@ -16,45 +11,17 @@ export function DrawerManageNetwork({
   setView,
   setSelectedNetwork
 }: DrawerManageNetworkProps) {
-  const {
-    state: { networks }
-  } = useGlobal()
   return (
-    <>
-      {networks.length ? (
-        <>
-          <Header style={{ marginTop: 0 }}>Networks</Header>
-          <BulletList style={{ marginBottom: 30 }}>
-            {networks.map(n => (
-              <BulletListItem key={n}>
-                <div
-                  style={{
-                    display: 'flex',
-                    justifyContent: 'space-between',
-                    alignItems: 'center'
-                  }}
-                >
-                  <span>{n}</span>
-                  <ButtonUnstyled
-                    data-testid='edit'
-                    onClick={() => {
-                      setSelectedNetwork(n)
-                      setView('edit')
-                    }}
-                    style={{ marginLeft: 'auto' }}
-                  >
-                    Edit
-                  </ButtonUnstyled>
-                </div>
-              </BulletListItem>
-            ))}
-          </BulletList>
-        </>
-      ) : null}
-      <Header style={{ marginTop: !networks.length ? 0 : undefined }}>
-        Add a network
-      </Header>
-      <NetworkImportForm />
-    </>
+    <div>
+      <NetworkPresets
+        setEditView={() => setView('edit')}
+        setSelectedNetwork={setSelectedNetwork}
+      />
+      <div style={{ margin: '24px 0' }}>
```

Comment:
suggestion(non-blocking): maybe use tailwind css? You can use custom/arbitrary values as well my-[24px]

Reply:
migrating the repo over to tailwind would be a massive task, and not related to the ticket.